workday/13.0/modules/ROOT/pages/workday-connector-examples.adoc

```diff
@@ -76,7 +76,6 @@ ns ns0 urn:com.workday/bsvc
 ----
 
 The following XML shows sample input to the *Transform Message* component
-*Michal, why are we showing this? Is this input sent with the body of the REST request?*:
```

Comment:
I remember there was one customer asking how the input looks like in XML format. Thats why I included exactly XML format.